packages/create-miniapp-cli/template/config/_eslint.drizzle.js

```
@@ -0,0 +1,58 @@
+
+export default tseslint.config(
+  {
+		ignores: ['.next']
+	},
+  ...compat.extends("next/core-web-vitals"),
+  {
+    files: ['**/*.ts', '**/*.tsx'],
+    plugins: {
+      drizzle,
+    },
+		extends: [
```

ESLint

[error] Expected indentation of 8 spaces but found 2 tabs.

(@typescript-eslint/indent)